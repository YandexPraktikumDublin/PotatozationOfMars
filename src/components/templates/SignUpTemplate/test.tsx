import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SignUpTemplate } from '.'

describe('<SignUpTemplate />', () => {
  it('should renders correct <SignUpTemplate />', () => {
    const wrapper = shallow(
      <SignUpTemplate title="test" text="test" path="text" />
    )

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
