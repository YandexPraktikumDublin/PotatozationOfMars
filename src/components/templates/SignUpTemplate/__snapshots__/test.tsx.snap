// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SignUpTemplate /> should renders correct <SignUpTemplate /> 1`] = `
<div
  className="w-full max-w-xs"
>
  <Window>
    <Title>
      test
    </Title>
    <SignUpForm />
    <FormLink
      className="inline-block mt-4"
      path="/auth/"
    >
      Already have an account?
    </FormLink>
  </Window>
</div>
`;
